src/pybind/mgr/dashboard/security.py

```diff
@@ -27,6 +27,7 @@ class Scope(object):
     DASHBOARD_SETTINGS = "dashboard-settings"
     NFS_GANESHA = "nfs-ganesha"
     NVME_OF = "nvme-of"
+    SMB = "smb"
```

Comment:
shall we also have the proper role defined for the smb management? something like

```
diff --git a/src/pybind/mgr/dashboard/services/access_control.py b/src/pybind/mgr/dashboard/services/access_control.py
index 21c1a9572bb..e8f150e038f 100644
--- a/src/pybind/mgr/dashboard/services/access_control.py
+++ b/src/pybind/mgr/dashboard/services/access_control.py
@@ -280,6 +280,13 @@ GANESHA_MGR_ROLE = Role(
         Scope.GRAFANA: [_P.READ],
     })
 
+SMB_MGR_ROLE = Role(
+    'smb-manager', 'allows full permissions for the smb scope', {
+        Scope.SMB: [_P.READ, _P.CREATE, _P.UPDATE, _P.DELETE],
+        Scope.CEPHFS: [_P.READ, _P.CREATE, _P.UPDATE, _P.DELETE],
+        Scope.GRAFANA: [_P.READ],
+    }
+)
 
 SYSTEM_ROLES = {
     ADMIN_ROLE.name: ADMIN_ROLE,
@@ -290,6 +297,7 @@ SYSTEM_ROLES = {
     POOL_MGR_ROLE.name: POOL_MGR_ROLE,
     CEPHFS_MGR_ROLE.name: CEPHFS_MGR_ROLE,
     GANESHA_MGR_ROLE.name: GANESHA_MGR_ROLE,
+    SMB_MGR_ROLE.name: SMB_MGR_ROLE
 }
 
```

Reply:
Sure, I can add it.

Do you think this new `SMB_MGR_ROLE` role should be shared with the `GANESHA_MGR_ROLE`? I think so right, since you don't want to be using same paths on the cephfs I think it makes sense to give access to both
What do you think @nizamial09 ?

Reply:
yup, sounds good.